morpheus/_lib/include/morpheus/stages/preprocess_nlp.hpp

```diff
@@ -54,6 +54,7 @@ class PreprocessNLPStage
                        bool truncation,
                        bool do_lower_case,
                        bool add_special_token,
+                       std::string column,
```

Comment:
Why does this come before `stride`? If it's due to the default value, we can use `"data"` for the default. If we change the order of the arguments, we need to mark this as breaking.

Can you move the `column` option to be after `stride` everywhere?

Reply:
sure

Reply:
@mdemoret-nv changed the order as suggested

Reply:
Great thanks.